crm/main.py

```diff
@@ -0,0 +1,489 @@
+        else:
+            data.append([resource, crm_info[resource + "_threshold_type"], crm_info[resource + "_low_threshold"], crm_info[resource + "_high_threshold"]])
+
+        print '\n'
+        print tabulate(data, headers=header, tablefmt="simple", missingval="")
+        print '\n'
+
+    def show_resources(self, resource):
+        """
+        CRM Handler to display resources information.
+        """
+        countersdb = swsssdk.SonicV2Connector(host='127.0.0.1')
+        countersdb.connect(countersdb.COUNTERS_DB)
+
+        crm_stats = countersdb.get_all(countersdb.COUNTERS_DB, 'CRM:STATS')
+
+        header = ("Resource Name", "Used Count", "Available Count")
+        data = []
+
+        if resource == 'all':
+            for res in ["ipv4_route", "ipv6_route", "ipv4_nexthop", "ipv6_nexthop", "ipv4_neighbor", "ipv6_neighbor",
+                        "nexthop_group_member", "nexthop_group", "fdb_entry"]:
+                data.append([res, crm_stats['crm_stats_' + res + "_used"], crm_stats['crm_stats_' + res + "_available"]])
+        else:
+            data.append([resource, crm_stats['crm_stats_' + resource + "_used"], crm_stats['crm_stats_' + resource + "_available"]])
+
+        print '\n'
+        print tabulate(data, headers=header, tablefmt="simple", missingval="")
+        print '\n'
+
+    def show_acl_resources(self):
+        """
+        CRM Handler to display ACL recources information.
+        """
+        countersdb = swsssdk.SonicV2Connector(host='127.0.0.1')
+        countersdb.connect(countersdb.COUNTERS_DB)
+
+        header = ("Stage", "Bind Point", "Resource Name", "Used Count", "Available Count")
+        data = []
+
+        for stage in ["INGRESS", "EGRESS"]:
+            for bind_point in ["PORT", "LAG", "VLAN", "RIF", "SWITCH"]:
+                crm_stats = countersdb.get_all(countersdb.COUNTERS_DB, 'CRM:ACL_STATS:{0}:{1}'.format(stage, bind_point))
+
+                if crm_stats:
+                    for res in ["acl_group", "acl_table"]:
+                        data.append([
+                                        stage, bind_point, res,
+                                        crm_stats['crm_stats_' + res + "_used"],
+                                        crm_stats['crm_stats_' + res + "_available"]
+                                    ])
+
+        print '\n'
+        print tabulate(data, headers=header, tablefmt="simple", missingval="")
+        print '\n'
+
+    def show_acl_table_resources(self):
+        """
+        CRM Handler to display ACL table information.
+        """
+        countersdb = swsssdk.SonicV2Connector(host='127.0.0.1')
+        countersdb.connect(countersdb.COUNTERS_DB)
+
+        header = ("Table ID", "Resource Name", "Used Count", "Available Count")
+
+        # Retrieve all ACL table keys from CRM:ACL_TABLE_STATS
+        proc = Popen("docker exec -i database redis-cli --raw -n 2 KEYS *CRM:ACL_TABLE_STATS*", stdout=PIPE, stderr=PIPE, shell=True)
```

Comment:
It exec another script to make DB operation. Could you please use redis-py or existing XxxxDBConnector/etc to do DB changes?